contracts/Quest.sol

```diff
@@ -105,18 +106,19 @@ contract Quest is OwnableUpgradeable, IQuest {
         if (tokens.length == 0) revert NoTokensToClaim();
 
         uint256 redeemableTokenCount = 0;
-        for (uint i = 0; i < tokens.length; i++) {
+        for (uint i = 0; i < tokens.length;) {
             if (!isClaimed(tokens[i])) {
-                redeemableTokenCount++;
+            unchecked{redeemableTokenCount++;}
```

Comment:
unsure about this one. I guess we should prob be safe for overflow here

Reply:
Yes, this is fine to me.